src/openrct2-android/app/src/main/java/io/openrct2/MainActivity.java

```diff
@@ -65,34 +68,8 @@ private PointF getResolutionDips() {
 
         DisplayMetrics metrics = new DisplayMetrics();
         display.getMetrics(metrics);
-
-        boolean success = false;
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```

Comment:
which version do we have as minimum now?

Reply:
We have the minimum set to 19, JELLY_BEAN_MR1 is 17